code/templates/index.html

```diff
@@ -45,10 +45,10 @@ <h4 class="p-text">A word we made up to describe how wonderful our burgers are.
                                         <div class="col-auto">
                                             <label class="p-text text-center" style="font-size:25pt;">Search Radius:</label>
                                             <select id="radius" class="form-control-sm" name="radius">
-                                                <option value="5">5</option>
                                                 <option value="10">10</option>
                                                 <option value="15">15</option>
                                                 <option value="20">20</option>
+                                                <option value="5">5</option>
```

Comment:
Use selected instead. https://stackoverflow.com/questions/3518002/how-can-i-set-the-default-value-for-an-html-select-element
```suggestion
                                                <option value="5" selected>5</option>
```